src/components/Agreements/ViewAgreement/ViewAgreement.js

```diff
@@ -376,6 +384,35 @@ class ViewAgreement extends React.Component {
     );
   }
 
+  renderDetailMenu(agreement) {
+    const {
+      tagsEnabled,
+      tagsToggle,
+    } = this.props;
+
+
+    const tags = ((agreement && agreement.tags) || {}) || [];
```

Comment:
This doesn't appear to make sense. The last false condition will never be executed.

Can you write these kinds of statements with `get` in the ERM apps instead? eg, `get(agreement, 'tags', [])`?

Reply:
Good catch. Will change it